config.json

```diff
@@ -53,11 +53,11 @@
 			"files": [
 				{
 					"destination": "scss/_colors.scss",
-					"format": "scss/variables"
+					"format": "scss/colorVariables"
 				}
 			]
 		},
-		"android": {
+		"androidColors": {
```

Comment:
Should we have MLC or another Android engineer about the Android color dist?

Reply:
this change won't affect the actual dist file for android - this is a configuration name only, to make it clear the "platform" target here is colors for android rather than all vars for andorid